src/frontend/src/eth/services/erc1155.services.ts

```diff
@@ -105,7 +107,10 @@ const loadCustomTokensWithMetadata = async (
 						standard: 'erc1155' as const,
 						category: 'custom' as const,
 						enabled,
-						version
+						version,
+						...(nonNullish(section) && {
+							section: 'Spam' in section ? CustomTokenSection.SPAM : CustomTokenSection.HIDDEN
```

Comment:
maybe just to be assertive?

```suggestion
							section: section === 'Spam' ? CustomTokenSection.SPAM : CustomTokenSection.HIDDEN
```